charts/sonarqube-dce/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sonarqube-dce
 description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
 type: application
-version: 5.1.2
+version: 5.1.3
```

Comment:
This is an improvement of the chart, as such we should upgrade the minor version